scripts/combined_commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+min_signal=${1:--1}
+dataroot=/data/datasets/LS4GAN_yuhw/full_images
+scriptroot=~/PROJs/LS4GAN/toytools/scripts/toyzero
+echo min signal = $min_signal
+read -n 1 -s -r -p "Press any key to continue"
+
+for sigproc in gauss orig
+do
+    echo signal processing = $sigproc
+    # We first generate a center crop of size 768x5888 from 800 x 6000 sized full frame
+    cmd_center_crop="python ${scriptroot}/center_crop ${dataroot}/${sigproc}/ ${dataroot}/${sigproc}_center_crop --plane U -s 768x5888"
+    # split the central-cuts into train and test
+    cmd_split="python ${scriptroot}/train_test_split ${dataroot}/${sigproc}_center_crop ${dataroot}/${sigproc}_split"
```

Comment:
Hi @pphuangyi. I think, you do not need to define strings and then call them as you do here. That is, for example this:
```bash
    cmd_center_crop="python ${scriptroot}/center_crop ${dataroot}/${sigproc}/ ${dataroot}/${sigproc}_center_crop --plane U -s 768x5888"
    $cmd_center_crop
```

can be simply written as:
```bash
    python ${scriptroot}/center_crop ${dataroot}/${sigproc}/ ${dataroot}/${sigproc}_center_crop --plane U -s 768x5888

```

Reply:
And same for the other lines

Reply:
But, I am fine if you leave it as is.